src/components/view/rocket/Rocket.js

```diff
@@ -0,0 +1,16 @@
+import { useSelector, useDispatch } from 'react-redux';
+import { useEffect } from 'react';
+import { GetRocketsFromApi } from '../../../redux/Rockets';
+
+const Rocket = () => {
+  useSelector((state) => state.rockets);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(GetRocketsFromApi());
+  }, [dispatch]);
+
+  return <div />;
+};
+
```

Comment:
please kindly render the required data here in the rockets component, so that you will see them in your UI. this is part of the project requirement. 🥇 👍